kornia/cli.py

```diff
@@ -0,0 +1,101 @@
+"""Command-line interface for Kornia."""
```

Comment:
personally, i would go with 

```console
foo@bar:/tmp/kornia $ tree kornia
kornia/
├── augmentation
│   ├── ...
├── ...
├── commands
│   ├── main.py // principal file with the principal CLI config
│   ├── ...
│   └── exporter.py // file with all export-related functions
├── ...
```

I can reorganize to have something like this above, in a further PR, if it makes sense for you

here is an example of it https://github.com/johnnv1/CCAgT-utils/tree/first-release/CCAgT_utils/commands

Reply:
yes, why not. My only question now is whether this should be packaged with the libirary, or where exactly should live, plus what's parts should be an api (like the KorniaModule that Jian suggested)